cypress/e2e/tests/migration/archetypes/crud.test.ts

```diff
@@ -85,4 +86,38 @@ describe(["@tier1"], "Archetype CRUD operations", () => {
         deleteByList(stakeholderGroups);
         deleteByList(tags);
     });
+
+    it("Duplicate archetype", function () {
+        // Automates Polarion MTA-399
+
+        const archetype = new Archetype(
+            data.getRandomWord(8),
+            [tags[0].name],
+            [tags[1].name],
+            null,
+            stakeholders,
+            stakeholderGroups
+        );
+        archetype.create();
+        checkSuccessAlert(
+            successAlertMessage,
+            `Success alert:Archetype ${archetype.name} was successfully created.`,
+            true
+        );
+        exists(archetype.name);
+
+        const archetypeDuplicate = archetype.duplicate();
+        checkSuccessAlert(
+            successAlertMessage,
+            `Success alert:Archetype ${archetypeDuplicate.name} was successfully created.`,
+            true
+        );
+        exists(archetypeDuplicate.name);
+
```

Comment:
Could you verify that the criteria and archetype tags of the original and duplicate architypes match ? 

I'm working on a PR that has an archetype method to open archetype side drawer panel. So, this verification could be done in a follow up PR .